codegen/templates/chart/operator-deployment.yamltmpl

```diff
@@ -12,8 +12,10 @@ Expressions evaluating SKv2 Config use [[ "[[" ]] and [[ "]]" ]]
   [[- $workloadKind = "DaemonSet" ]]
   [[- end -]]
 [[- $operatorVar := (lower_camel $operator.Name) -]]
-[[- $extraPodLabels := $operator.Deployment.Labels -]]
-[[- $extraPodAnnotations := $operator.Deployment.Annotations -]]
+[[- $extraPodLabels := $operator.Deployment.PodLabels -]]
```

Comment:
Same comment here, though more nitpicky, can we use the same name for the template variable as we do for the struct field?